### ISSUE TITLE
Installation woes: `numpy.ndarray size changed`

### ISSUE THREAD

**Issue by arosen93** [hspark1212/MOFTransformer#75]
Hello! Very nice code. I am trying to install it but am having some issues with the requirements. 

I am on Windows 11 using Anaconda with a fresh Python 3.8 env. I pip install'd pytorch and then pip installed the current commit ([6a42856](https://github.com/hspark1212/MOFTransformer/commit/6a4285618998ab8594b3445a37c353d1a229b575)) of MOFTransformer. Calling `moftransformer` from the command line then yields the following:

```
Traceback (most recent call last):
  File "C:\Users\asros\miniconda\envs\test\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\asros\miniconda\envs\test\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\asros\miniconda\envs\test\Scripts\moftransformer.exe\__main__.py", line 4, in <module>
  File "C:\Users\asros\miniconda\envs\test\lib\site-packages\moftransformer\__init__.py", line 6, in <module>
    from moftransformer import visualize, utils, modules, libs, gadgets, datamodules, assets
  File "C:\Users\asros\miniconda\envs\test\lib\site-packages\moftransformer\visualize\__init__.py", line 1, in <module>
    from moftransformer.visualize.visualizer import PatchVisualizer
  File "C:\Users\asros\miniconda\envs\test\lib\site-packages\moftransformer\visualize\visualizer.py", line 11, in <module>
    from moftransformer.visualize.utils import get_structure, get_heatmap, scaler, get_model_and_datamodule, \
  File "C:\Users\asros\miniconda\envs\test\lib\site-packages\moftransformer\visualize\utils.py", line 10, in <module>
    from pymatgen.io.cif import CifParser
  File "C:\Users\asros\miniconda\envs\test\lib\site-packages\pymatgen\io\cif.py", line 25, in <module>
    from pymatgen.core.composition import Composition
  File "C:\Users\asros\miniconda\envs\test\lib\site-packages\pymatgen\core\__init__.py", line 18, in <module>
    from .lattice import Lattice as Lattice
  File "C:\Users\asros\miniconda\envs\test\lib\site-packages\pymatgen\core\lattice.py", line 24, in <module>
    from pymatgen.util.coord import pbc_shortest_vectors
  File "C:\Users\asros\miniconda\envs\test\lib\site-packages\pymatgen\util\coord.py", line 18, in <module>
    from pymatgen.util import coord_cython as cuc
  File "pymatgen\util\coord_cython.pyx", line 1, in init pymatgen.util.coord_cython
ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
```

Any ideas?

Additional details:
```
(test) λ pip install torch
Collecting torch
  Downloading torch-1.13.0-cp38-cp38-win_amd64.whl (167.3 MB)
     ------------------------ 167.3/167.3 MB 13.1 MB/s eta 0:00:00
Collecting typing-extensions
  Using cached typing_extensions-4.4.0-py3-none-any.whl (26 kB)
Installing collected packages: typing-extensions, torch
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
scvelo 0.2.4 requires matplotlib>=3.3.0, which is not installed.
scvelo 0.2.4 requires pandas>=0.23, which is not installed.
scvelo 0.2.4 requires scikit-learn>=0.21.2, which is not installed.
scvelo 0.2.4 requires scipy>=1.4.1, which is not installed.
cellrank 1.5.1 requires joblib>=0.13.1, which is not installed.
cellrank 1.5.1 requires matplotlib>=3.3.0, which is not installed.
cellrank 1.5.1 requires networkx>=2.2, which is not installed.
cellrank 1.5.1 requires pandas>=1.2.0, which is not installed.
cellrank 1.5.1 requires scikit-learn>=0.24.0, which is not installed.
cellrank 1.5.1 requires scipy>=1.2.0, which is not installed.
cellrank 1.5.1 requires wrapt>=1.12.1, which is not installed.
Successfully installed torch-1.13.0 typing-extensions-4.4.0

(test) λ pip install git+https://github.com/hspark1212/MOFTransform.git
Collecting git+https://github.com/hspark1212/MOFTransformer.git
  Cloning https://github.com/hspark1212/MOFTransformer.git to c:\users\asros\appdata\local\temp\pip-req-build-rhew1957
  Running command git clone --filter=blob:none --quiet https://github.com/hspark1212/MOFTransformer.git 'C:\Users\asros\AppData\Local\Temp\pip-req-build-rhew1957'
  Resolved https://github.com/hspark1212/MOFTransformer.git to commit 6a4285618998ab8594b3445a37c353d1a229b575
  Preparing metadata (setup.py) ... done
Collecting wget
  Downloading wget-3.2.zip (10 kB)
  Preparing metadata (setup.py) ... done
Collecting pytorch-lightning>=1.8.0
  Downloading pytorch_lightning-1.8.3.post0-py3-none-any.whl (798 kB)
     ------------------------- 799.0/799.0 kB 7.2 MB/s eta 0:00:00
Collecting torchmetrics>=0.6.0
  Downloading torchmetrics-0.10.3-py3-none-any.whl (529 kB)
     -------------------------------- 529.7/529.7 kB ? eta 0:00:00
Collecting transformers>=4.12.5
  Downloading transformers-4.24.0-py3-none-any.whl (5.5 MB)
     ---------------------------- 5.5/5.5 MB 21.9 MB/s eta 0:00:00
Collecting timm>=0.4.12
  Downloading timm-0.6.12-py3-none-any.whl (549 kB)
     ------------------------ 549.1/549.1 kB 35.9 MB/s eta 0:00:00
Collecting sacred>=0.8.2
  Downloading sacred-0.8.2-py2.py3-none-any.whl (106 kB)
     -------------------------------- 106.0/106.0 kB ? eta 0:00:00
Collecting einops>=0.4.1
  Downloading einops-0.6.0-py3-none-any.whl (41 kB)
     ---------------------------------- 41.6/41.6 kB ? eta 0:00:00
Collecting ase>=3.22.1
  Using cached ase-3.22.1-py3-none-any.whl (2.2 MB)
Collecting pymatgen>=2022.0.16
  Downloading pymatgen-2022.11.7-cp38-cp38-win_amd64.whl (3.0 MB)
     ---------------------------- 3.0/3.0 MB 32.3 MB/s eta 0:00:00
Collecting matplotlib>=3.5.0
  Downloading matplotlib-3.6.2-cp38-cp38-win_amd64.whl (7.2 MB)
     ---------------------------- 7.2/7.2 MB 30.8 MB/s eta 0:00:00
Collecting seaborn>=0.12.0
  Downloading seaborn-0.12.1-py3-none-any.whl (288 kB)
     -------------------------------- 288.2/288.2 kB ? eta 0:00:00
Collecting ipympl>=0.9.2
  Downloading ipympl-0.9.2-py2.py3-none-any.whl (510 kB)
     ------------------------ 510.3/510.3 kB 31.3 MB/s eta 0:00:00
Collecting pandas
  Downloading pandas-1.5.2-cp38-cp38-win_amd64.whl (11.0 MB)
     -------------------------- 11.0/11.0 MB 21.1 MB/s eta 0:00:00
Collecting tqdm
  Using cached tqdm-4.64.1-py2.py3-none-any.whl (78 kB)
Collecting google-auth
  Downloading google_auth-2.14.1-py2.py3-none-any.whl (175 kB)
     -------------------------------- 175.4/175.4 kB ? eta 0:00:00
Collecting jupyter-client
  Downloading jupyter_client-7.4.7-py3-none-any.whl (133 kB)
     ------------------------- 133.5/133.5 kB 7.7 MB/s eta 0:00:00
Collecting smmap<6
  Using cached smmap-5.0.0-py3-none-any.whl (24 kB)
Collecting emmet-core
  Downloading emmet_core-0.38.9-py3-none-any.whl (232 kB)
     -------------------------------- 232.8/232.8 kB ? eta 0:00:00
Requirement already satisfied: numpy in c:\users\asros\appdata\roaming\python\python38\site-packages (from moftransformer==1.1.0) (1.21.0)
Collecting scipy>=1.1.0
  Downloading scipy-1.9.3-cp38-cp38-win_amd64.whl (39.8 MB)
     -------------------------- 39.8/39.8 MB 24.2 MB/s eta 0:00:00
Collecting ipython<9
  Downloading ipython-8.6.0-py3-none-any.whl (761 kB)
     ------------------------ 761.1/761.1 kB 50.1 MB/s eta 0:00:00
Collecting ipython-genutils
  Using cached ipython_genutils-0.2.0-py2.py3-none-any.whl (26 kB)
Collecting traitlets<6
  Using cached traitlets-5.5.0-py3-none-any.whl (107 kB)
Collecting pillow
  Downloading Pillow-9.3.0-cp38-cp38-win_amd64.whl (2.5 MB)
     ---------------------------- 2.5/2.5 MB 31.8 MB/s eta 0:00:00
Collecting ipywidgets<9,>=7.6.0
  Downloading ipywidgets-8.0.2-py3-none-any.whl (134 kB)
     -------------------------------- 134.4/134.4 kB ? eta 0:00:00
Collecting packaging>=20.0
  Using cached packaging-21.3-py3-none-any.whl (40 kB)
Collecting fonttools>=4.22.0
  Downloading fonttools-4.38.0-py3-none-any.whl (965 kB)
     ------------------------ 965.4/965.4 kB 30.8 MB/s eta 0:00:00
Collecting python-dateutil>=2.7
  Using cached python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
Collecting cycler>=0.10
  Using cached cycler-0.11.0-py3-none-any.whl (6.4 kB)
Collecting kiwisolver>=1.0.1
  Downloading kiwisolver-1.4.4-cp38-cp38-win_amd64.whl (55 kB)
     ---------------------------------- 55.4/55.4 kB ? eta 0:00:00
Collecting pyparsing>=2.2.1
  Using cached pyparsing-3.0.9-py3-none-any.whl (98 kB)
Collecting contourpy>=1.0.1
  Downloading contourpy-1.0.6-cp38-cp38-win_amd64.whl (163 kB)
     -------------------------------- 163.5/163.5 kB ? eta 0:00:00
Collecting tabulate
  Using cached tabulate-0.9.0-py3-none-any.whl (35 kB)
Collecting pybtex
  Using cached pybtex-0.24.0-py2.py3-none-any.whl (561 kB)
Collecting requests
  Using cached requests-2.28.1-py3-none-any.whl (62 kB)
Collecting sympy
  Using cached sympy-1.11.1-py3-none-any.whl (6.5 MB)
Collecting mp-api>=0.27.3
  Downloading mp_api-0.29.8-py3-none-any.whl (70 kB)
     ---------------------------------- 70.6/70.6 kB ? eta 0:00:00
Collecting monty>=3.0.2
  Using cached monty-2022.9.9-py3-none-any.whl (66 kB)
Collecting plotly>=4.5.0
  Using cached plotly-5.11.0-py2.py3-none-any.whl (15.3 MB)
Collecting palettable>=3.1.1
  Using cached palettable-3.3.0-py2.py3-none-any.whl (111 kB)
Collecting ruamel.yaml>=0.17.0
  Using cached ruamel.yaml-0.17.21-py3-none-any.whl (109 kB)
Collecting spglib>=2.0.2
  Downloading spglib-2.0.2-cp38-cp38-win_amd64.whl (289 kB)
     ------------------------ 289.3/289.3 kB 17.4 MB/s eta 0:00:00
Collecting networkx>=2.2
  Downloading networkx-2.8.8-py3-none-any.whl (2.0 MB)
     ---------------------------- 2.0/2.0 MB 32.0 MB/s eta 0:00:00
Collecting uncertainties>=3.1.4
  Using cached uncertainties-3.1.7-py2.py3-none-any.whl (98 kB)
Collecting fsspec[http]>2021.06.0
  Downloading fsspec-2022.11.0-py3-none-any.whl (139 kB)
     -------------------------------- 139.5/139.5 kB ? eta 0:00:00
Requirement already satisfied: typing-extensions>=4.0.0 in c:\users\asros\miniconda\envs\test\lib\site-packages (from pytorch-lightning>=1.8.0->moftransformer==1.1.0) (4.4.0)
Collecting lightning-utilities==0.3.*
  Downloading lightning_utilities-0.3.0-py3-none-any.whl (15 kB)
Collecting tensorboardX>=2.2
  Downloading tensorboardX-2.5.1-py2.py3-none-any.whl (125 kB)
     ------------------------- 125.4/125.4 kB 7.2 MB/s eta 0:00:00
Requirement already satisfied: torch>=1.9.* in c:\users\asros\miniconda\envs\test\lib\site-packages (from pytorch-lightning>=1.8.0->moftransformer==1.1.0) (1.13.0)
Collecting PyYAML>=5.4
  Using cached PyYAML-6.0-cp38-cp38-win_amd64.whl (155 kB)
Collecting fire
  Downloading fire-0.4.0.tar.gz (87 kB)
     ---------------------------------- 87.7/87.7 kB ? eta 0:00:00
  Preparing metadata (setup.py) ... done
Collecting py-cpuinfo>=4.0
  Downloading py_cpuinfo-9.0.0-py3-none-any.whl (22 kB)
Collecting wrapt<2.0,>=1.0
  Using cached wrapt-1.14.1-cp38-cp38-win_amd64.whl (35 kB)
Collecting docopt<1.0,>=0.3
  Downloading docopt-0.6.2.tar.gz (25 kB)
  Preparing metadata (setup.py) ... done
Collecting colorama>=0.4
  Using cached colorama-0.4.6-py2.py3-none-any.whl (25 kB)
Collecting GitPython
  Downloading GitPython-3.1.29-py3-none-any.whl (182 kB)
     ------------------------ 182.5/182.5 kB 11.5 MB/s eta 0:00:00
Collecting munch<3.0,>=2.0.2
  Downloading munch-2.5.0-py2.py3-none-any.whl (10 kB)
Collecting jsonpickle<2.0,>=1.2
  Downloading jsonpickle-1.5.2-py2.py3-none-any.whl (37 kB)
Collecting pytz>=2020.1
  Downloading pytz-2022.6-py2.py3-none-any.whl (498 kB)
     ------------------------ 498.1/498.1 kB 15.7 MB/s eta 0:00:00
Collecting huggingface-hub
  Downloading huggingface_hub-0.11.0-py3-none-any.whl (182 kB)
     -------------------------------- 182.1/182.1 kB ? eta 0:00:00
Collecting torchvision
  Downloading torchvision-0.14.0-cp38-cp38-win_amd64.whl (1.1 MB)
     ---------------------------- 1.1/1.1 MB 35.1 MB/s eta 0:00:00
Collecting filelock
  Downloading filelock-3.8.0-py3-none-any.whl (10 kB)
Collecting tokenizers!=0.11.3,<0.14,>=0.11.1
  Downloading tokenizers-0.13.2-cp38-cp38-win_amd64.whl (3.3 MB)
     ---------------------------- 3.3/3.3 MB 26.3 MB/s eta 0:00:00
Collecting regex!=2019.12.17
  Downloading regex-2022.10.31-cp38-cp38-win_amd64.whl (267 kB)
     -------------------------------- 267.7/267.7 kB ? eta 0:00:00
Collecting pydantic>=1.10.2
  Downloading pydantic-1.10.2-cp38-cp38-win_amd64.whl (2.2 MB)
     ---------------------------- 2.2/2.2 MB 27.7 MB/s eta 0:00:00
Collecting cachetools<6.0,>=2.0.0
  Using cached cachetools-5.2.0-py3-none-any.whl (9.3 kB)
Collecting six>=1.9.0
  Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting pyasn1-modules>=0.2.1
  Using cached pyasn1_modules-0.2.8-py2.py3-none-any.whl (155 kB)
Collecting rsa<5,>=3.1.4
  Downloading rsa-4.9-py3-none-any.whl (34 kB)
Collecting entrypoints
  Using cached entrypoints-0.4-py3-none-any.whl (5.3 kB)
Collecting tornado>=6.2
  Using cached tornado-6.2-cp37-abi3-win_amd64.whl (425 kB)
Collecting nest-asyncio>=1.5.4
  Using cached nest_asyncio-1.5.6-py3-none-any.whl (5.2 kB)
Collecting jupyter-core>=4.9.2
  Downloading jupyter_core-5.0.0-py3-none-any.whl (91 kB)
     --------------------------- 91.1/91.1 kB 5.1 MB/s eta 0:00:00
Collecting pyzmq>=23.0
  Downloading pyzmq-24.0.1-cp38-cp38-win_amd64.whl (998 kB)
     ------------------------ 999.0/999.0 kB 21.0 MB/s eta 0:00:00
Collecting aiohttp!=4.0.0a0,!=4.0.0a1
  Downloading aiohttp-3.8.3-cp38-cp38-win_amd64.whl (324 kB)
     ------------------------ 324.3/324.3 kB 20.9 MB/s eta 0:00:00
Collecting prompt-toolkit<3.1.0,>3.0.1
  Downloading prompt_toolkit-3.0.33-py3-none-any.whl (383 kB)
     ------------------------ 383.6/383.6 kB 24.9 MB/s eta 0:00:00
Collecting backcall
  Using cached backcall-0.2.0-py2.py3-none-any.whl (11 kB)
Collecting jedi>=0.16
  Downloading jedi-0.18.2-py2.py3-none-any.whl (1.6 MB)
     ---------------------------- 1.6/1.6 MB 33.9 MB/s eta 0:00:00
Collecting matplotlib-inline
  Using cached matplotlib_inline-0.1.6-py3-none-any.whl (9.4 kB)
Collecting decorator
  Using cached decorator-5.1.1-py3-none-any.whl (9.1 kB)
Collecting pickleshare
  Using cached pickleshare-0.7.5-py2.py3-none-any.whl (6.9 kB)
Collecting stack-data
  Downloading stack_data-0.6.1-py3-none-any.whl (24 kB)
Collecting pygments>=2.4.0
  Using cached Pygments-2.13.0-py3-none-any.whl (1.1 MB)
Collecting ipykernel>=4.5.1
  Downloading ipykernel-6.17.1-py3-none-any.whl (138 kB)
     -------------------------------- 138.6/138.6 kB ? eta 0:00:00
Collecting jupyterlab-widgets~=3.0
  Downloading jupyterlab_widgets-3.0.3-py3-none-any.whl (384 kB)
     -------------------------------- 384.1/384.1 kB ? eta 0:00:00
Collecting widgetsnbextension~=4.0
  Downloading widgetsnbextension-4.0.3-py3-none-any.whl (2.0 MB)
     ---------------------------- 2.0/2.0 MB 26.0 MB/s eta 0:00:00
Collecting platformdirs
  Downloading platformdirs-2.5.4-py3-none-any.whl (14 kB)
Collecting pywin32>=1.0
  Downloading pywin32-305-cp38-cp38-win_amd64.whl (12.3 MB)
     -------------------------- 12.3/12.3 MB 21.8 MB/s eta 0:00:00
Requirement already satisfied: setuptools in c:\users\asros\miniconda\envs\test\lib\site-packages (from mp-api>=0.27.3->pymatgen>=2022.0.16->moftransformer==1.1.0) (65.5.1)
Collecting msgpack
  Using cached msgpack-1.0.4-cp38-cp38-win_amd64.whl (62 kB)
Collecting tenacity>=6.2.0
  Using cached tenacity-8.1.0-py3-none-any.whl (23 kB)
Collecting pyasn1<0.5.0,>=0.4.6
  Using cached pyasn1-0.4.8-py2.py3-none-any.whl (77 kB)
Collecting latexcodec>=1.0.4
  Using cached latexcodec-2.0.1-py2.py3-none-any.whl (18 kB)
Collecting certifi>=2017.4.17
  Using cached certifi-2022.9.24-py3-none-any.whl (161 kB)
Collecting idna<4,>=2.5
  Using cached idna-3.4-py3-none-any.whl (61 kB)
Collecting urllib3<1.27,>=1.21.1
  Downloading urllib3-1.26.13-py2.py3-none-any.whl (140 kB)
     -------------------------------- 140.6/140.6 kB ? eta 0:00:00
Collecting charset-normalizer<3,>=2
  Using cached charset_normalizer-2.1.1-py3-none-any.whl (39 kB)
Collecting ruamel.yaml.clib>=0.2.6
  Downloading ruamel.yaml.clib-0.2.7-cp38-cp38-win_amd64.whl (118 kB)
     -------------------------------- 118.0/118.0 kB ? eta 0:00:00
Collecting protobuf<=3.20.1,>=3.8.0
  Using cached protobuf-3.20.1-cp38-cp38-win_amd64.whl (904 kB)
Collecting future
  Using cached future-0.18.2.tar.gz (829 kB)
  Preparing metadata (setup.py) ... done
Collecting gitdb<5,>=4.0.1
  Using cached gitdb-4.0.9-py3-none-any.whl (63 kB)
Collecting mpmath>=0.19
  Using cached mpmath-1.2.1-py3-none-any.whl (532 kB)
Collecting yarl<2.0,>=1.0
  Downloading yarl-1.8.1-cp38-cp38-win_amd64.whl (56 kB)
     --------------------------- 56.8/56.8 kB 3.1 MB/s eta 0:00:00
Collecting frozenlist>=1.1.1
  Downloading frozenlist-1.3.3-cp38-cp38-win_amd64.whl (34 kB)
Requirement already satisfied: attrs>=17.3.0 in c:\users\asros\appdata\roaming\python\python38\site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->fsspec[http]>2021.06.0->pytorch-lightning>=1.8.0->moftransformer==1.1.0) (21.4.0)
Collecting async-timeout<5.0,>=4.0.0a3
  Using cached async_timeout-4.0.2-py3-none-any.whl (5.8 kB)
Collecting multidict<7.0,>=4.5
  Using cached multidict-6.0.2-cp38-cp38-win_amd64.whl (28 kB)
Collecting aiosignal>=1.1.2
  Downloading aiosignal-1.3.1-py3-none-any.whl (7.6 kB)
Collecting psutil
  Downloading psutil-5.9.4-cp36-abi3-win_amd64.whl (252 kB)
     ------------------------ 252.5/252.5 kB 16.1 MB/s eta 0:00:00
Collecting debugpy>=1.0
  Downloading debugpy-1.6.3-cp38-cp38-win_amd64.whl (4.6 MB)
     ---------------------------- 4.6/4.6 MB 24.5 MB/s eta 0:00:00
Collecting parso<0.9.0,>=0.8.0
  Using cached parso-0.8.3-py2.py3-none-any.whl (100 kB)
Collecting wcwidth
  Using cached wcwidth-0.2.5-py2.py3-none-any.whl (30 kB)
Collecting termcolor
  Downloading termcolor-2.1.1-py3-none-any.whl (6.2 kB)
Collecting pure-eval
  Using cached pure_eval-0.2.2-py3-none-any.whl (11 kB)
Collecting asttokens>=2.1.0
  Downloading asttokens-2.1.0-py2.py3-none-any.whl (26 kB)
Collecting executing>=1.2.0
  Downloading executing-1.2.0-py2.py3-none-any.whl (24 kB)
Building wheels for collected packages: moftransformer, wget, docopt, fire, future
  Building wheel for moftransformer (setup.py) ... done
  Created wheel for moftransformer: filename=moftransformer-1.1.0-py3-none-any.whl size=1905788 sha256=4d4b31ddc30802f2ba9ce293c248bdb8fb724f113341f16a81d4c054f24dd927
  Stored in directory: C:\Users\asros\AppData\Local\Temp\pip-ephem-wheel-cache-0781dxz_\wheels\07\cd\fc\c9cc5a905e01d10e597ae89d2d3bf9aba9ddf4ad0b9eba964d
  Building wheel for wget (setup.py) ... done
  Created wheel for wget: filename=wget-3.2-py3-none-any.whl size=9680 sha256=0ba2b7e70a3c21f613af596f6affeaeed6338714c0ddd6d63555ce01dbd0347a
  Stored in directory: c:\users\asros\appdata\local\pip\cache\wheels\46\4a\43\6e71c9584e8b20b326931deaf89f11bf4db1bda170ac38e7b4
  Building wheel for docopt (setup.py) ... done
  Created wheel for docopt: filename=docopt-0.6.2-py2.py3-none-any.whl size=13775 sha256=5b0dbbe52e4a80afcbf31f9ffbaf165ef6f8692a83a3be9946bb40585798b18a
  Stored in directory: c:\users\asros\appdata\local\pip\cache\wheels\ca\cc\e3\f1e272f628fdb013d969acc99cfe2e031ea15b3efb74ffe842
  Building wheel for fire (setup.py) ... done
  Created wheel for fire: filename=fire-0.4.0-py2.py3-none-any.whl size=115943 sha256=24fcd71ef410db78a499af195f73289beb09aba0303daed5662b84d8e43c0d60
  Stored in directory: c:\users\asros\appdata\local\pip\cache\wheels\61\05\8d\5951e074fe660f634ca0a3402fa9903d9f772014fdb0e593dd
  Building wheel for future (setup.py) ... done
  Created wheel for future: filename=future-0.18.2-py3-none-any.whl size=491086 sha256=aff933204cfd8ec8373c8657a3b2797fcc62a4ee32a1ff155b6fadc8d5d27f45
  Stored in directory: c:\users\asros\appdata\local\pip\cache\wheels\01\49\0c\4e0a697824c7bd6516afb22e1af9d51427ccd36c74b05a297e
Successfully built moftransformer wget docopt fire future
Installing collected packages: wget, wcwidth, tokenizers, pywin32, pytz, pyasn1, py-cpuinfo, pure-eval, pickleshare, palettable, msgpack, mpmath, ipython-genutils, executing, docopt, backcall, wrapt, widgetsnbextension, urllib3, traitlets, tornado, termcolor, tenacity, tabulate, sympy, spglib, smmap, six, scipy, ruamel.yaml.clib, rsa, regex, pyzmq, PyYAML, pyparsing, pygments, pydantic, pyasn1-modules, psutil, protobuf, prompt-toolkit, platformdirs, pillow, parso, networkx, nest-asyncio, multidict, monty, kiwisolver, jupyterlab-widgets, jsonpickle, idna, future, fsspec, frozenlist, fonttools, filelock, entrypoints, einops, decorator, debugpy, cycler, contourpy, colorama, charset-normalizer, certifi, cachetools, async-timeout, yarl, uncertainties, tqdm, tensorboardX, ruamel.yaml, requests, python-dateutil, plotly, packaging, munch, matplotlib-inline, latexcodec, jupyter-core, jedi, google-auth, gitdb, fire, asttokens, aiosignal, torchvision, torchmetrics, stack-data, pybtex, pandas, matplotlib, lightning-utilities, jupyter-client, huggingface-hub, GitPython, aiohttp, transformers, timm, seaborn, sacred, ipython, ase, pytorch-lightning, ipykernel, ipywidgets, ipympl, mp-api, emmet-core, pymatgen, moftransformer
  Attempting uninstall: seaborn
    Found existing installation: seaborn 0.11.2
    Uninstalling seaborn-0.11.2:
      Successfully uninstalled seaborn-0.11.2
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
umap-learn 0.5.2 requires scikit-learn>=0.22, which is not installed.
scvelo 0.2.4 requires scikit-learn>=0.21.2, which is not installed.
scanpy 1.8.2 requires h5py>=2.10.0, which is not installed.
scanpy 1.8.2 requires joblib, which is not installed.
scanpy 1.8.2 requires natsort, which is not installed.
scanpy 1.8.2 requires scikit-learn>=0.22, which is not installed.
scanpy 1.8.2 requires sinfo, which is not installed.
pynndescent 0.5.6 requires joblib>=0.11, which is not installed.
pynndescent 0.5.6 requires llvmlite>=0.30, which is not installed.
pynndescent 0.5.6 requires scikit-learn>=0.18, which is not installed.
loompy 3.0.7 requires click, which is not installed.
loompy 3.0.7 requires h5py, which is not installed.
loompy 3.0.7 requires numpy-groupies, which is not installed.
cellrank 1.5.1 requires joblib>=0.13.1, which is not installed.
cellrank 1.5.1 requires scikit-learn>=0.24.0, which is not installed.
anndata 0.8.0 requires h5py>=3, which is not installed.
anndata 0.8.0 requires natsort, which is not installed.
Successfully installed GitPython-3.1.29 PyYAML-6.0 aiohttp-3.8.3 aiosignal-1.3.1 ase-3.22.1 asttokens-2.1.0 async-timeout-4.0.2 backcall-0.2.0 cachetools-5.2.0 certifi-2022.9.24 charset-normalizer-2.1.1 colorama-0.4.6 contourpy-1.0.6 cycler-0.11.0 debugpy-1.6.3 decorator-5.1.1 docopt-0.6.2 einops-0.6.0 emmet-core-0.38.9 entrypoints-0.4 executing-1.2.0 filelock-3.8.0 fire-0.4.0 fonttools-4.38.0 frozenlist-1.3.3 fsspec-2022.11.0 future-0.18.2 gitdb-4.0.9 google-auth-2.14.1 huggingface-hub-0.11.0 idna-3.4 ipykernel-6.17.1 ipympl-0.9.2 ipython-8.6.0 ipython-genutils-0.2.0 ipywidgets-8.0.2 jedi-0.18.2 jsonpickle-1.5.2 jupyter-client-7.4.7 jupyter-core-5.0.0 jupyterlab-widgets-3.0.3 kiwisolver-1.4.4 latexcodec-2.0.1 lightning-utilities-0.3.0 matplotlib-3.6.2 matplotlib-inline-0.1.6 moftransformer-1.1.0 monty-2022.9.9 mp-api-0.29.8 mpmath-1.2.1 msgpack-1.0.4 multidict-6.0.2 munch-2.5.0 nest-asyncio-1.5.6 networkx-2.8.8 packaging-21.3 palettable-3.3.0 pandas-1.5.2 parso-0.8.3 pickleshare-0.7.5 pillow-9.3.0 platformdirs-2.5.4 plotly-5.11.0 prompt-toolkit-3.0.33 protobuf-3.20.1 psutil-5.9.4 pure-eval-0.2.2 py-cpuinfo-9.0.0 pyasn1-0.4.8 pyasn1-modules-0.2.8 pybtex-0.24.0 pydantic-1.10.2 pygments-2.13.0 pymatgen-2022.11.7 pyparsing-3.0.9 python-dateutil-2.8.2 pytorch-lightning-1.8.3.post0 pytz-2022.6 pywin32-305 pyzmq-24.0.1 regex-2022.10.31 requests-2.28.1 rsa-4.9 ruamel.yaml-0.17.21 ruamel.yaml.clib-0.2.7 sacred-0.8.2 scipy-1.9.3 seaborn-0.12.1 six-1.16.0 smmap-5.0.0 spglib-2.0.2 stack-data-0.6.1 sympy-1.11.1 tabulate-0.9.0 tenacity-8.1.0 tensorboardX-2.5.1 termcolor-2.1.1 timm-0.6.12 tokenizers-0.13.2 torchmetrics-0.10.3 torchvision-0.14.0 tornado-6.2 tqdm-4.64.1 traitlets-5.5.0 transformers-4.24.0 uncertainties-3.1.7 urllib3-1.26.13 wcwidth-0.2.5 wget-3.2 widgetsnbextension-4.0.3 wrapt-1.14.1 yarl-1.8.1

(test) λ moftransformer download hmof
Traceback (most recent call last):
  File "C:\Users\asros\miniconda\envs\test\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\asros\miniconda\envs\test\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\asros\miniconda\envs\test\Scripts\moftransformer.exe\__main__.py", line 4, in <module>
  File "C:\Users\asros\miniconda\envs\test\lib\site-packages\moftransformer\__init__.py", line 6, in <module>
    from moftransformer import visualize, utils, modules, libs, gadgets, datamodules, assets
  File "C:\Users\asros\miniconda\envs\test\lib\site-packages\moftransformer\visualize\__init__.py", line 1, in <module>
    from moftransformer.visualize.visualizer import PatchVisualizer
  File "C:\Users\asros\miniconda\envs\test\lib\site-packages\moftransformer\visualize\visualizer.py", line 11, in <module>
    from moftransformer.visualize.utils import get_structure, get_heatmap, scaler, get_model_and_datamodule, \
  File "C:\Users\asros\miniconda\envs\test\lib\site-packages\moftransformer\visualize\utils.py", line 10, in <module>
    from pymatgen.io.cif import CifParser
  File "C:\Users\asros\miniconda\envs\test\lib\site-packages\pymatgen\io\cif.py", line 25, in <module>
    from pymatgen.core.composition import Composition
  File "C:\Users\asros\miniconda\envs\test\lib\site-packages\pymatgen\core\__init__.py", line 18, in <module>
    from .lattice import Lattice as Lattice
  File "C:\Users\asros\miniconda\envs\test\lib\site-packages\pymatgen\core\lattice.py", line 24, in <module>
    from pymatgen.util.coord import pbc_shortest_vectors
  File "C:\Users\asros\miniconda\envs\test\lib\site-packages\pymatgen\util\coord.py", line 18, in <module>
    from pymatgen.util import coord_cython as cuc
  File "pymatgen\util\coord_cython.pyx", line 1, in init pymatgen.util.coord_cython
ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
```

**Comment by hspark1212** [comment]
Nice to see you here !

We realized that the numpy version is not compatible with pymatgen. So the error would be fixed if you upgrade numpy (>= 1.23.2).
And we just released moftransformer=1.1.1 where some bugs regarding installation were fixed, you can use it !

Thank you for your comment and let us know if there is an additional question.

**Comment by arosen93** [comment]
Great, thank you!
